### PR TITLE
 Add numismatics fixtures and coin fields

### PR DESCRIPTION
**Pull request by eliotjordan**
Basic coin display.

Advances #1567

### CODE REVIEW

File: app/assets/stylesheets/icons/icons.scss [hunk 1]
```diff
@@ -182,6 +182,13 @@
         content: $icon-lp;
     }
 }
+.icon-coin,
+.blacklight-coin .document-thumbnail .default {
+    &:before {
+        content: $icon-coins-stack;
+        // content: $icon-coins-flat;
```
Comment (jrgriffiniii) [inline app/assets/stylesheets/icons/icons.scss]: This comment was not removed after being disabled.

Comment (eliotjordan) [reply app/assets/stylesheets/icons/icons.scss]: Fixed.

File: app/controllers/catalog_controller.rb [hunk 2]
```diff
@@ -293,6 +293,52 @@ class CatalogController < ApplicationController
...
+    config.add_show_field 'issue_reverse_orientation_t', label: 'Reverse Orientation'
+    config.add_show_field 'issue_reverse_part_t', label: 'Reverse Part'
+    config.add_show_field 'issue_reverse_symbol_t', label: 'Reverse Symbol'
+    config.add_show_field 'issue_ruler_t', label: 'Ruler'
+    config.add_show_field 'issue_series_t', label: 'Series'
+    config.add_show_field 'issue_shape_t', label: 'Shape'
+    config.add_show_field 'issue_workshop_t', label: 'Workshop'
+    config.add_show_field 'size_t', label: 'Size'
+    config.add_show_field 'technique_t', label: 'Technique'
+    config.add_show_field 'title_display', label: 'Title'
```
Comment (christinach) [inline app/controllers/catalog_controller.rb]: title_display , field is already used in the header display

Comment (eliotjordan) [reply app/controllers/catalog_controller.rb]: @christinach Fixed.

**Pull request merged**